Tests/Test_BSP_Contactors.c

```diff
@@ -0,0 +1,12 @@
+#include "common.h"
+#include "config.h"
+#include <bsp.h>
+
+int main() {
+    BSP_Contactors_Init();
+    BSP_Contactor_Set(1,0);
+    int16_t states[5];
+    read_Contactor_States(states);
+    printf("%d%d", states[0], states[1]);
+    exit(0);
```

Comment:
I would prefer it if this test was built to interact with the Python side. Something in an infinite loop that continuously asks for you to change the states or something. Based on user input.

Reply:
Test now has an infinite loop that accepts user input in the command line. I tested it and it worked without error for me. I wasn't quite sure if this test file was what you're looking for, so let me know if I need to make any changes to it so that it's more useful/helpful!